### PR TITLE
Bug in Logo Rendering in Light mode

### PR DESCRIPTION
**Pull request by ashwath462**
Logo Bug
[Link to issue this resolves](https://github.com/ashwath462/eui-event-template)

Here is the screenshot of Bug
bug

 The title should be something descriptive about what you're changes do. (It will default to whatever you put as your commit message.)
 changed the icon rendering
 https://github.com/ashwath462/eui-event-template
 Screen shot of the image given below
resolved

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { EuiIcon, EuiPageHeader } from "@elastic/eui";
 import { addCalButtons } from "../utilities/addCalButtons";
 /*import { makeRainbowText } from "../utilities/rainbowLetters";*/
-import { sessionOne, sessionTwo, eventIcon } from "../data/consts";
+import { sessionOne, sessionTwo, DarkThemeLogo, LightThemeLogo } from "../data/consts";
```
Comment (brittanyjoiner15) [inline src/components/Navbar.jsx]:
```suggestion
import { sessionOne, sessionTwo, darkThemeLogo, lightThemeLogo } from "../data/consts";
```

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default function Navbar(props) {
     <>
       <EuiPageHeader
         restrictWidth
-        iconType={eventIcon}
+        iconType={props.theme === "dark"?DarkThemeLogo:LightThemeLogo}
```
Comment (brittanyjoiner15) [inline src/components/Navbar.jsx]:
```suggestion
        iconType={props.theme === "dark"? darkThemeLogo : lightThemeLogo}
```

File: src/data/consts.js [hunk 3]
```diff
@@ -10,4 +11,5 @@ export const sessionTwo = {
   calendarLink: "#",
 };
 
-export const eventIcon = logo
\ No newline at end of file
+export const DarkThemeLogo = logo
```
Comment (brittanyjoiner15) [inline src/data/consts.js]:
```suggestion
export const darkThemeLogo = logo
```

File: src/data/consts.js [hunk 4]
```diff
@@ -10,4 +11,5 @@ export const sessionTwo = {
...
+export const LightThemeLogo = logo2
```
Comment (brittanyjoiner15) [inline src/data/consts.js]:
```suggestion
export const lightThemeLogo = logo2
```

**Review by brittanyjoiner15 — CHANGES_REQUESTED** [review]
Quick fix - just need to make sure it starts with a lowercase letter as we prefer to name variables with`camelCase`. You should be able to just click "commit suggestion" and it should implement those changes, then I can approve. Just let me know once you've done that! Also i will close out your other PR.

**Comment by ashwath462** [comment]
> Quick fix - just need to make sure it starts with a lowercase letter as we prefer to name variables with`camelCase`. You should be able to just click "commit suggestion" and it should implement those changes, then I can approve. Just let me know once you've done that! Also i will close out your other PR.

Mam i have done the sugested changes please look into it. Thank you.

**Review by brittanyjoiner15 — APPROVED** [review]
This looks great! Thanks @ashwath462 for your work on this!

**Pull request merged**